<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerWidget</class>
 <widget class="QWidget" name="PlayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Player</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>3</number>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <widget class="TeamRadar::TeamRadarView" name="graphicsView"/>
       </item>
       <item>
        <widget class="QWidget" name="widgetControls" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelConnection">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="Resources.qrc">:/Images/Red.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btPlayPause">
            <property name="toolTip">
             <string>Play/Pause</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="sbSpeed">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Speed</string>
            </property>
            <property name="suffix">
             <string> X</string>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btLoad">
            <property name="toolTip">
             <string>Load</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btDownload">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/Images/Download.png</normaloff>:/Images/Download.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btOnline">
            <property name="toolTip">
             <string>Online/Offline</string>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/Images/Connect.png</normaloff>:/Images/Connect.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btPlaylist">
            <property name="toolTip">
             <string>Playlist</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btEffects">
            <property name="toolTip">
             <string>Use Effects</string>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/Images/Shadow.png</normaloff>:/Images/Shadow.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btAnalyze">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/Images/Analyze.png</normaloff>:/Images/Analyze.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableView" name="tvPlaylist">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TeamRadar::TeamRadarView</class>
   <extends>QGraphicsView</extends>
   <header>TeamRadarView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
